lib/notifications/receive.dart

```diff
@@ -149,8 +149,8 @@ class NotificationService {
         await addFcmToken(connection, token: token);
 
       case TargetPlatform.iOS:
-        const appBundleId = 'com.zulip.flutter'; // TODO(#407) find actual value live
-        await addApnsToken(connection, token: token, appid: appBundleId);
+         // TODO(#407) find actual value live
+        await addApnsToken(connection, token: token, appid: (await ZulipBinding.instance.packageInfo)?.packageName ?? "com.zulip.flutter");
```

Comment:
I think there is an underlying issue if `packageInfo` is missing, so let's include a `// TODO(log)` comment here.

Reply:
This case is interesting so it might worth a test for it. Let's find a way to test when we fallback to this default value, if `packageInfo` is somehow missing.

Reply:
So, should we test for packageInfo being null just incase of packageName?, because packageInfo also have other members like build number and version.

Reply:
`packageInfo` itself is nullable, so it makes sense to test that.